src/llamator/client/chat_client.py

```diff
@@ -206,26 +209,32 @@ def __init__(
         stop_criterion : Optional[Callable[[List[Dict[str, str]]], bool]], optional
             A function that takes the tested client's history and returns True if the conversation should stop.
             If None, a default criterion that always returns False is used. (default is None)
-        refine_attack_prompt : Optional[Callable[..., str]], optional
+        refine_tested_client_prompt : Optional[Callable[..., str]], optional
```

Comment:
`refine_tested_client_prompt`  ---> `score_tested_client_prompt`

returns `float`, not `str`